docs/pages/guides/monorepos.md

```diff
@@ -70,32 +70,122 @@ Metro doesn't come with monorepo support by default (yet). That's why we need to
+
+When using monorepos, your app dependencies are split up into different directories. Each of these directories must be within the scope of the [watchFolders](https://facebook.github.io/metro/docs/configuration/#watchfolders). If a file is outside of that scope, Metro won't be able to find it. Setting this path to the root of your monorepo
+
+As your monorepo increases in size, watching all files within the monorepo becomes slower. If you want to speed that up, you can (dynamically) watch the relevant paths to minimize the watch scope. For example:
+
+```js
+// Learn more https://docs.expo.dev/guides/monorepos
+const { getDefaultConfig } = require('expo/metro-config');
+const path = require('path');
+
+const projectRoot = __dirname;
+const workspaceRoot = path.resolve(projectRoot, '../..');
+
+// If your monorepo tooling can give you the list of monorepo workspaces linked 
+// in your app workspace, you can automate this list instead of hardcoding them.
+const monorepoPackages = {
+  '@acme/api': path.resolve(workspaceRoot, 'packages/api'), 
+  '@acme/components': path.resolve(workspaceRoot, 'packages/components'),
+};
+
+// 1. Watch the local app folder, and all related packages (limiting the scope and speeding it up)
+config.watchFolders = [__dirname, ...Object.values(monorepoPackages)];
+// Add the monorepo workspaces as `extraNodeModules` to Metro.
+// If your monorepo tooling creates workspace symlinks in the `node_modules` folder,
+// you can either add symlink support to Metro or set the `extraNodeModules` to avoid the symlinks.
+// See: https://facebook.github.io/metro/docs/configuration/#extranodemodules
+config.resolver.extraNodeModules = monorepoPackages;
+
+// 2. Let Metro know where to resolve packages and in what order
+config.resolver.nodeModulesPaths = [
+  path.resolve(projectRoot, 'node_modules'),
+  path.resolve(workspaceRoot, 'node_modules'),
+];
+// 3. Force Metro to resolve (sub)dependencies only from the `nodeModulesPaths`
+config.resolver.disableHierarchicalLookup = true;
+```
+
+</Collapsible>
+
+<Collapsible summary="2. Why do we need tell Metro how to resolve packages?">
+
+This option is important to resolve libraries in the correct **node_modules** directories. Monorepo tooling, like Yarn, usually creates two different **node_modules** directories which are used for a single workspace.
+
+1. **apps/mobile/node_modules** - The "project" folder
+2. **node_modules** - The "root" folder
+
+Yarn uses the root folder to install packages that are used in multiple workspaces. If a workspace uses a different package version, it installs that different version in the project folder.
```

Comment:
```suggestion
Yarn uses the root folder to install packages used in multiple workspaces. If a workspace uses a different package version, it installs that different version in the project folder.
```

Suggestion to avoid passive voice usage.